### DISCUSSION TITLE
能否为 transform 的文档，补充使用场景说明和一些 demo 演示？

### DISCUSSION THREAD

**Discussion by pfdgithub** [antvis/G2#5621] [Q&A] [answered]
现在对于 transform 中的各种类型，很多只有一句简单的介绍，对于不熟悉 d3 的用户，很多名词都不清楚含义。

一般情况下，用户是为了实现某种图表中的某种效果，才去查找 API 使用文档。但现在的文档编写方式，是某个 API 有哪些参数，最多再放一个单一的使用场景。
参数说明非常简短，没有 demo 几乎无法理解含义。单一的使用场景演示，又造成无法理解 API 的可以适用于哪些其它场景。

**Comment by pearmini** [comment, marked as answer]
这里有一个真实的分析案例：https://observablehq.com/@antv/g2-exploration-athletes-of-asian-games-2023

分析了亚运会的运动员，用了大量 transform API。有帮助的话也可以点一个赞。
